README.md

```diff
@@ -1,89 +1,75 @@
-WoX
-===
+Flow Launcher
+=============
 
 ![Maintenance](https://img.shields.io/maintenance/yes/2020)
-[![GitHub release (latest by date)](https://img.shields.io/github/v/release/jjw24/wox)](https://github.com/jjw24/Flow.Launcher/releases/latest)
-![GitHub Release Date](https://img.shields.io/github/release-date/jjw24/wox)
-![GitHub commits since latest release](https://img.shields.io/github/commits-since/jjw24/wox/v1.3.524)
-[![Build Status](https://dev.azure.com/Flow.Launcher-Launcher/Flow.Launcher/_apis/build/status/jjw24.Flow.Launcher?branchName=master)](https://dev.azure.com/Flow.Launcher-Launcher/Flow.Launcher/_build/latest?definitionId=1&branchName=master)
-[![Github All Releases](https://img.shields.io/github/downloads/jjw24/Flow.Launcher/total.svg)](https://github.com/jjw24/Flow.Launcher/releases)
-[![RamenBless](https://cdn.rawgit.com/LunaGao/BlessYourCodeTag/master/tags/ramen.svg)](https://github.com/LunaGao/BlessYourCodeTag)
+[![Github All Releases](https://img.shields.io/github/downloads/Flow-Launcher/Flow.Launcher/total.svg)](https://github.com/Flow-Launcher/Flow.Launcher/releases)
+[![GitHub release (latest by date)](https://img.shields.io/github/v/release/Flow-Launcher/Flow.Launcher)](https://github.com/Flow-Launcher/Flow.Launcher/releases/latest)
+![GitHub Release Date](https://img.shields.io/github/release-date/Flow-Launcher/Flow.Launcher)
+[![Build Status](https://dev.azure.com/Flow-Launcher/Flow.Launcher/_apis/build/status/Flow.Launcher?branchName=master)](https://dev.azure.com/Flow-Launcher/Flow.Launcher/_build/latest?definitionId=1&branchName=master)
 
-**WoX** is a launcher for Windows that simply works. It's an alternative to [Alfred](https://www.alfredapp.com/) and [Launchy](http://www.launchy.net/). You can call it Windows omni-eXecutor if you want a long name.
+Nickname flow, full name Flow Launcher. Dedicated to make your workflow flow more seamlessly. Aimed at being more than an app launcher, it searches, integrates and expands on functionalities. Flow will continue to evolve, designed to be open and built with the community at heart.
```

Comment:
I'd make this sentence simpler:

```suggestion
Flow Launcher. Dedicated to make your workflow flow more seamlessly. Aimed at being more than an app launcher, it searches, integrates and expands on functionalities. Flow will continue to evolve, designed to be open and built with the community at heart.
```

Reply:
Haha I was trying to sound a bit playful with it, not feeling it?

Also that we can refer to it as Flow for short if we want to later on.

Reply:
At first I liked it, but I think that sounding impersonal might give it a better feel :)